core/ir/opnd_shared.c

```diff
@@ -2775,7 +2779,7 @@ reg_get_size(reg_id_t reg)
         return OPSZ_8;
     if (reg == DR_REG_WZR)
         return OPSZ_4;
-    if (reg >= DR_REG_MDCCSR_EL0 && reg <= DR_REG_SPSR_FIQ)
+    if (reg >= DR_REG_NZCV && reg <= DR_REG_SPSR_FIQ)
```

Comment:
This is unrelated to the described change: could be separated out; at least mention in PR description.

Reply:
But it is unrelated to #7199 or adding DR_REG_AFTER_LAST_VALID_ENUM and normally wouldn't be bundled together with it: always best to keep separate changes separate for easier reverting, code understanding, etc.  That said, it is small, and there is overhead in creating and submitting separate PRs, so maybe it's ok to keep bundled in here: but please mention it in the PR description as otherwise it looks weird and the reader puzzles over how this is related to this PR.